src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -393,6 +393,8 @@ Running your own validator allows for [solo staking](https://ethereum.org/en/sta
 
 The easiest way to get started with staking and validator key generation is to use the [Prater Testnet Staking Launchpad](https://prater.launchpad.ethereum.org/), which allows you to test your setup by [running nodes on Goerli](https://notes.ethereum.org/@launchpad/goerli). When you're ready for Mainnet, you can repeat these steps using the [Mainnet Staking Launchpad](https://launchpad.ethereum.org/). Make sure to check [Mainnet readiness checklist](https://launchpad.ethereum.org/en/merge-readiness) to smoothly run your validator through the Merge.
 
+Look into [aditional resources](#further-reading) for more guides on staking. 
```

Comment:
Shouldn't we direct people to our /staking/ pages here instead?

Meta: let's ensure we're conscious that this page is mostly run-a-node and mostly not run-a-validator.

Reply:
Agreed, I tried to keep validator stuff on minimum. I will change the link to https://ethereum.org/en/staking/